openmmtools/multistate/utils.py

```diff
@@ -177,12 +178,14 @@ def get_equilibration_data_per_sample(timeseries_to_analyze, fast=True, max_subs
     i_t = np.floor(counter * time_size / max_subset).astype(int)
     for i, t in enumerate(i_t):
         try:
-            g_i[i] = timeseries.statisticalInefficiency(series[t:], fast=fast)
-        except:
+            g_i[i] = statistical_inefficiency(series[t:], fast=fast)
+        except Exception as e:
+            print("If you see this, open an issue https://github.com/choderalab/openmmtools/issues")
+            raise e
```

Comment:
Do we want to raise the exception here? I feel like this might be disruptive for current users/applications.

Reply:
See https://github.com/choderalab/openmmtools/pull/659#discussion_r1138117826 I think we can see if perses has any problems as well as other downstream packages as a double check. I worry it does more damage leaving it in the codebase. I could be convinced to just log an error but I think no one will see it.